include/wx/clrpicker.h

```diff
@@ -159,13 +161,15 @@ class WXDLLIMPEXP_CORE wxColourPickerCtrl : public wxPickerBase
 // ----------------------------------------------------------------------------
 
 wxDECLARE_EXPORTED_EVENT( WXDLLIMPEXP_CORE, wxEVT_COLOURPICKER_CHANGED, wxColourPickerEvent );
+wxDECLARE_EXPORTED_EVENT( WXDLLIMPEXP_CORE, wxEVT_COLOUR_SELECTED, wxColourPickerEvent );
+wxDECLARE_EXPORTED_EVENT( WXDLLIMPEXP_CORE, wxEVT_COLOUR_CANCELED, wxColourPickerEvent );
```

Comment:
I believe that wxWidgets use spelling with two Ls, i.e., "canceled" should be changed to "cancelled" everywhere.

Reply:
Thanks i will fix it :)